v2/tools/generator/internal/astmodel/property_definition_test.go

```diff
@@ -515,8 +517,8 @@ func Test_PropertyDefinitionAsAst_GivenValidField_ReturnsNonNilResult(t *testing
 		MakeRequired().
 		WithDescription(propertyDescription)
 
-	node := original.AsField(nil)
-
+	node, err := original.AsField(nil)
+	g.Expect(err).To(BeNil())
```

Comment:
`ToNot(HaveOccurred())` is I think more idiomatic as far as gomega is concerned and (I think) gives a better error message if there is an assertion failure.

Reply:
Changed.